scripts/peekJob.pl

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env perl
+
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2019] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+    # Finding out own path in order to reference own components (including own modules):
+use Cwd            ();
+use File::Basename ();
+BEGIN {
+    $ENV{'EHIVE_ROOT_DIR'} ||= File::Basename::dirname( File::Basename::dirname( Cwd::realpath($0) ) );
+    unshift @INC, $ENV{'EHIVE_ROOT_DIR'}.'/modules';
+}
+
+use Getopt::Long qw(:config no_auto_abbrev);
+use Pod::Usage;
+
+use Bio::EnsEMBL::Hive::Version qw(report_versions);
+use Bio::EnsEMBL::Hive::HivePipeline;
+use Bio::EnsEMBL::Hive::Scripts::PeekJob;
+use Bio::EnsEMBL::Hive::Utils::URL;
+
+Bio::EnsEMBL::Hive::Utils::URL::hide_url_password();
+
+main();
+
+
+sub main {
+    my ($url, $reg_conf, $reg_type, $reg_alias, $nosqlvc, $job_id); 
+    my ($help, $report_versions);
+
+    $|=1;   # make STDOUT unbuffered (STDERR is unbuffered anyway)
+
+    GetOptions(
+
+    # Connection parameters:
+               'url=s'                        => \$url,
+               'reg_conf|regfile|reg_file=s'  => \$reg_conf,
+               'reg_type=s'                   => \$reg_type,
+               'reg_alias|regname|reg_name=s' => \$reg_alias,
+               'nosqlvc'                      => \$nosqlvc,       # using "nosqlvc" instead of "sqlvc!" for consistency with scripts where it is a propagated option
+
+    # Task specification parameters:
+               'job_id=i'                   => \$job_id,
+
+    # Other commands
+               'h|help'                     => \$help,
+               'v|version|versions'         => \$report_versions,
```

Comment:
The `-version` option is not implemented in all scripts. Mainly just runWorker and beekeeper. You can remove it from this script